src/main/java/com/acwong/peelplaylistcheck/playlist/PlayListHelper.java

```diff
@@ -0,0 +1,22 @@
+package com.acwong.peelplaylistcheck.playlist;
+
+import java.io.File;
+
+public interface PlayListHelper {
```

Comment:
What's the point of this class? Why not just merge it with Impl?

Reply:
Can you come up with a better abstaction than Helper?

Reply:
I started from building the unit test first and so using the interface can be easier, and allow possible growth of the software in case we have different strategies to get playlists.

So I think there are two ways.  Either we foresee this potential extension upfront, or we start without it (as there is no other strategy so far and we may not have another strategy requiring switching.  If there will be, we could still extend/refactor accordingly when needed).

I could do another update to consolidate into just one PlayListHelper once all other comments are agreed.

---

On an alternative approach with another abstraction, this can be to refactor the methods into the associated object classes.  i.e. 

* all methods from PlayListHelperImpl can be moved to ContentCollection (a derived class can be used if a clear separation from JSON object is of higher importance).  For this update, we could have them directly in the ContentCollection.

Do you have another better suggestion that I could consider?  Will do update once agreed.

Reply:
Good points. Yes, in general I think a class named "Helper" usually indicates a weak abstraction that hasn't been fully thought out. One exception is if it is implementing ViewHelper pattern.

Reply:
ContentCollection may be a reasonable place for these methods.